docs/release-notes/master-docinfo.xml

```diff
@@ -7,7 +7,7 @@
         {ProductName} {DocInfoProductNumber} accelerates large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift via the user interface.
     </para>
 	<para>
-		This document describes new features and improvements, known issues, and resolved issues for the {DocInfoProductName}, version 6.1.0.
+		This document describes new features and improvements, known issues, and resolved issues for the {DocInfoProductName}, version {DocInfoProductNumber}.
```

Comment:
@sbeskin-redhat - let us change it to {DocInfoProductNumber} and then this will not trip us next time